environments/values-usdfdev.yaml

```diff
@@ -19,7 +19,7 @@ gafaelfawr:
 hips:
   enabled: false
 ingress-nginx:
-  enabled: true
+  enabled: false
```

Comment:
This does have to be enabled. We don't support a configuration without ingress-nginx enabled. We can still merge this as a temporary step towards more convergence, but wanted to flag that this will have to change back in the future.

Reply:
we are running [loft.sh's vcluster](https://github.com/loft-sh/vcluster/tree/main) and have an ingress-nginx installed in the 'parent' cluster. we are using vcluster in the mode where we run phalanx inside a virtualised cluster and (local) Ingress requests get translated up to the 'parent' - as such an ingress-nginx inside the phalanx'd vcluster isn't necessary and will use the global one that we have installed. this also allows us to apply kyverno policies to limit hijacking of ingress `host:` etc. we have been running like this since we first installed phalanx.

Reply:
yep, understood. i did wonder why you created a GF Ingress object type and would like to understand further the specific requirements and implementation details. we (un?)fortunately have to implement some degree of security controls on our side too, and this was deemed a good compromise.

Reply:
Rationale for `GafaelfawrIngress` is here: https://dmtn-193.lsst.io/#configuration

Reply:
That at least should be unrelated to this, though, since `GafaelfawrIngress` is processed by the Gafaelfawr Kubernetes operator into normal `Ingress` objects. The tricky part is that they contain a reference to a handler location that needs to be injected into every server in the generated NGINX configuration, which is necessary to get auth subrequests to work properly.